data_models/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Run this script with 'sudo -u postgres install.sh'
+
+# This script defaults to postgres version 9.5.
+# To use a different version set the POSTGRES_VERSION env var
+# (i.e. 'sudo POSTGRES_VERSION=9.4 -u postgres install.sh')
+if [ -z $POSTGRES_VERSION ]; then POSTGRES_VERSION=9.5; fi
+
+createdb bioinformatics_mill
+
+psql bioinformatics_mill -c "CREATE ROLE bioinformatics_mill_user WITH LOGIN PASSWORD 'bioinformatics_password';"
```

Comment:
I think if it's really true that this script won't be used in production then the risk is relatively low, but my concern highlighted above about *accidentally* leaving this in place still applies. At the very least, I think we'd need a big noisy warning in the comments above and written to the output of this command to the effect that this is **NOT FOR PRODUCTION** to mitigate that risk.